pythonforandroid/recipe.py

```diff
@@ -817,24 +816,55 @@ def install_python_package(self, arch, name=None, env=None, is_dir=True):
                 self.install_hostpython_package(arch)
 
     def get_hostrecipe_env(self, arch):
-        env = environ.copy()
-        env['PYTHONPATH'] = join(dirname(self.real_hostpython_location), 'Lib', 'site-packages')
+        env = {}
+        env['PATH'] = '/usr/local/bin:/usr/bin:/bin'
+        env['PYTHONPATH'] = join(dirname(self.real_hostpython_location),
+                                 'Lib', 'site-packages')
         return env
 
     def install_hostpython_package(self, arch):
+
+        info('install_hostpython_package: {}'.format(self.name))
+
         env = self.get_hostrecipe_env(arch)
         real_hostpython = sh.Command(self.real_hostpython_location)
         shprint(real_hostpython, 'setup.py', 'install', '-O2',
                 '--root={}'.format(dirname(self.real_hostpython_location)),
                 '--install-lib=Lib/site-packages',
                 _env=env, *self.setup_extra_args)
 
+    def pip_install_hostpython_package(self, arch):
```

Comment:
Are you aware of the existing `install_in_hostpython` and `install_in_targetpython` PythonRecipe options? Is there a reason for not using these (making any improvements if necessary) instead of this method? Even if that does need some changes, I think that would be a preferable API.

Reply:
According to the pull request description `hostpython now uses host_pip to speed up building`.